### PR TITLE
WIP-Structure-Built

### PR DESCRIPTION
**Pull request by PengyuW007**

### CODE REVIEW

**Comment by CZUBRYTT** [comment]
The main code looks good so far.

**Review by CZUBRYTT — APPROVED** [review]
The main code looks good so far.

**Review by PengyuW007 — COMMENTED** [review]
Looks so good Martin.

**Review by Zhiyi233 — APPROVED** [review]
The main.py looks pretty nice. Good job Martin!

**Review by PengyuW007 — COMMENTED** [review]
Please add all the packages into README, or if you don't know how to modify README file, please tell me then I'll add them later.

File: README.md [hunk 1]
```diff
@@ -6,8 +6,8 @@
 1. Pengyu Wang, 7863401
 2. Thomas Czubryt, 
 3. Hantong Li(Lucas),
```
Comment (PengyuW007) [inline README.md]: Please add your student ID @CZUBRYTT and @Lucasbeast

**Review by PengyuW007 — COMMENTED** [review]
Can you make some comments, which follows programming standard, to make code more readable? If markers doesn't know what are the meanings of anywhere, then he can check comments, that would be nicer.

**Comment by CZUBRYTT** [comment]
Great work so far!
Based on the code, the PER_ASM column in probability.csv refers to the percentage of people who are SYMPTOMATIC, not asymptomatic, since the code uses `data1 = testFile.query('COV_ASM == 2')`. If you want the percentage of people who are ASYMPTOMATIC, you need to do `data1 = testFile.query('COV_ASM == 1')`. Also, the PER_DTH column refers to the percentage of people who have NOT died, since the code uses `data2 = testFile.query('COV_DTH == 2')`. If you want the percentage of people who HAVE died, you need to do `data2 = testFile.query('COV_DTH == 1')`. Also, the PER_HSP column refers to the percentage of people who have been hospitalized AND have been in intensive care. I'm not sure whether or not whether or not you also want to include people who have been hospitalized but are not in intensive care, but if you want to, you'll need to do something like `data0 = testFile.query('COV_HSP <= 2')`.

**Comment by Zhiyi233** [comment]
> 

Thomas, Lucas has relied your question on the Slack.

**Comment by Zhiyi233** [comment]
> Great work so far! Based on the code, the PER_ASM column in probability.csv refers to the percentage of people who are SYMPTOMATIC, not asymptomatic, since the code uses `data1 = testFile.query('COV_ASM == 2')`. If you want the percentage of people who are ASYMPTOMATIC, you need to do `data1 = testFile.query('COV_ASM == 1')`. Also, the PER_DTH column refers to the percentage of people who have NOT died, since the code uses `data2 = testFile.query('COV_DTH == 2')`. If you want the percentage of people who HAVE died, you need to do `data2 = testFile.query('COV_DTH == 1')`. Also, the PER_HSP column refers to the percentage of people who have been hospitalized AND have been in intensive care. I'm not sure whether or not whether or not you also want to include people who have been hospitalized but are not in intensive care, but if you want to, you'll need to do something like `data0 = testFile.query('COV_HSP <= 2')`.

Lucas has replied your question on the Slack.

**Pull request merged**